### PR TITLE
fix(badge): fix weird badge styling

### PR DESCRIPTION
**Pull request by t-howell**
Attempting to help with issue #581 - Sorry about the commit message format, it was my first try at angular formatting, I'll have to practice

### CODE REVIEW

**Comment by alejandronanez** [comment]
@t-howell can you please upload a screenshot of the end result?

Thanks!

**Comment by housseindjirdeh** [comment]
Thank you @t-howell <3

And no worries at all matey! We'll squash them anyway as we merge them in. Seconding @alejandronanez and could you please upload a before and after screenshot when you get the chance?

**Comment by jouderianjr** [comment]
Hey, also the text color should be white, that was removed after the refactor to style components :/

File: src/components/badge.component.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const BadgeContainer = styled.View`
   flex: 1;
   align-items: center;
   justify-content: center;
-  border-radius: 18;
+  border-radius: 25;
```
Comment (machour) [inline src/components/badge.component.js]: normally, in order to have a circle, the border-radius should be equal to width & height.
could you provide a screenshot of the result?
don't hesitate if you need any assistance with this PR!

**Comment by machour** [comment]
@t-howell I looked at the badge again this morning and I didn't notice the weird borders anymore. I guess this was somehow fixed by upgrading to latest react-native-version. 🤔

Could you rollback the `border-radius` change and make the text white so that we can merge this PR for the 1.4.0 release?

Let me know if you need any assistance. 🙏

**Comment by t-howell** [comment]
Hey all, sorry for the late reply! I didn't expect to hear back on this one, honestly. This was my first go at contributing to a project - I'm a newb.

I couldn't get the environment running, so this was just my best guess. What do you mean by "rollback"? I can definitely make the text white.

**Comment by machour** [comment]
@t-howell Thank you for your honesty and no worries mate! We all had to start somewhere 😉 

To rollback something is to cancel the changes made (as opposed to roll out a new feature I guess)
So the idea would be to put back `border-radius: 18` and just take care of making the text white.

We do however require you to provide a screenshot in your PR, so you will need to get the project running on your side. If you're planning to do more react/react-native development in the future, that could be a good occasion to get you started. 

Do you want to go over our [setup instructions](https://github.com/gitpoint/git-point/blob/master/CONTRIBUTING.md#setup) and try to do that? We're available on [gitter](https://gitter.im/git-point/Lobby) if you get stuck somewhere.

**Comment by machour** [comment]
@t-howell are you still up for this? We need to fix this bug for the upcoming 1.4 release 🙏 
I can take care of it if you don't have time right now, and will be more than happy to guide you through your next PR!

**Comment by t-howell** [comment]
Would you take care of it? I'd love help figuring out this React thing when things are less busy! Thanks for your understanding.

**Comment by machour** [comment]
@t-howell no worries at all mate 🙌
We'll be more than happy to help you get up and running when you have free time again, don't hesitate then!

Closing this PR, in favor of #648

**Pull request closed without merging**